src/licence-status-banner/index.jsx

```diff
@@ -71,7 +71,7 @@ class LicenceStatusBanner extends Component {
           {
             isInactiveVersion && <Fragment>
               <p><Snippet>invalidLicence.summary.ppl_active</Snippet></p>
-              <p><Link page="project.version.read" versionId={this.props.licence.granted.id} label={<Snippet>{'invalidLicence.view'}</Snippet>} /></p>
+              <p><Link page="projectVersion" versionId={this.props.licence.granted.id} label={<Snippet>{'invalidLicence.view'}</Snippet>} /></p>
```

Comment:
We should define this component in `@asl/pages` rather than here.

(It can stay here but unchanged for now, and then be removed in due course once all the references are gone)

Reply:
👍